app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False):
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+    def hide(self):
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(animal):
+        if isinstance(animal, Herbivore) and animal.hidden is False:
+            animal.health -= 50
+            if not animal.health:
```

Comment:
What if `animal.health` will become below 0?

Reply:
I refactored my code and pulled another version of it. Don't know why it didn't upload.

Reply:
@Yurnerroo Check your branches. You created this pull request from `develop1` branch.